packages/plugin/src/theia.d.ts

```diff
@@ -2871,6 +2871,70 @@ declare module '@theia/plugin' {
         setDimensions?(dimensions: TerminalDimensions): void;
     }
 
+    /**
+     * Provides information on a line in a terminal in order to provide links for it.
+     */
+    // copied from https://github.com/microsoft/vscode/blob/bd20a720fba05f87ddb53c11c6af66936fd88a55/src/vs/vscode.d.ts#L5563-L5576
```

Comment:
i think you should remove all the `// copied` comments. this whole file is based on vscode.d.ts